models/issue_milestone.go

```diff
@@ -465,3 +493,83 @@ func DeleteMilestoneByRepoID(repoID, id int64) error {
 	}
 	return sess.Commit()
 }
+
+// CountMilestonesByRepo map from repoIDs to number of milestones matching the options`
+func CountMilestonesByRepo(repoIDs []int64, isClosed bool) (map[int64]int64, error) {
```

Comment:
Smal nit: this func consume RepoIDs but only next function has ByID in name ...

Reply:
Edit: current existing func already mix this :(

Reply:
I think that's a fair point, I changed to "CountMilestonesByRepoIDs" so the naming matches up with the next function "GetMilestonesByRepoIDs"

Reply:
looks good, I'll try it later :)